pages/docs/running/orchestration.mdx

```diff
@@ -0,0 +1,161 @@
+  },
+];
+```
+
+This method is the simplest and is built into Grouparoo directly.
+
+## Triggering Schedules via the API
+
+Using the Grouparoo API, you can choose to run a [Schedule](/docs/getting-started/product-concepts#schedule) whenever it is right for you. A common use-case is to trigger all of your Grouparoo Schedules to look for new & updated Records at the conclusion of a [dbt](https://www.getdbt.com/) transform step via and Orchestrator like [Airflow](https://airflow.apache.org/) or [Dagster](https://dagster.io/).
+
+The `/api/v1/schedules/run` endpoint can be used to trigger a Run for your Schedules. You can run all of your Schedules or provide a list of specific `scheduleIds` to run.
+
+First, create an [API Key](/docs/getting-started/product-concepts#api-key) with the `write` permission on `Sources`, and then you can use the API as documented below:
+
+```bash
+# Set Variables
+GROUPAROO_HOST="http://localhost:3000"
+GROUPAROO_API_KEY="ba05b18d9dae4acbb08799e7ef238f0d"
+
+# Request to run all Schedules now
+curl \
+  -X POST \
+  -d "apiKey=$GROUPAROO_API_KEY" \
+"$GROUPAROO_HOST/api/v1/schedules/run"
+
+# --- OR ---
+
+# Request to view the available schedules
+curl \
+  -X GET \
+"$GROUPAROO_HOST/api/v1/schedules?apiKey=$GROUPAROO_API_KEY"
+
+# Request to run a specific schedule now
+# The names of your schedules will be of the format (source_name)_schedule
+# Provide IDs as a JSON Array
+curl \
+  -X POST \
+  -d "apiKey=$GROUPAROO_API_KEY" \
+  -d "scheduleIds=[\"users_table_schedule\"]" \
+"$GROUPAROO_HOST/api/v1/schedules/run"
+```
+
+More information about using [Grouparoo's REST API can be found here](/docs/support/rest-api), including how to generate API Keys. You can learn more about the specifics of this (or any) API endpoint that your Grouparoo instance provides by visiting the `/swagger` page on your Grouparoo instance.
+
+If you want to wait until the Runs created by the call to `/api/v1/schedules/run` are complete before moving on to the next step in your data pipeline, you can! You can also use the API to retrieve run status:
+
+```bash
+# Store the Run ID
+RUN_ID="run_aad727ae-b8f8-4a6a-b205-015113bf0e7c"
+
+# Request details of a Run
+# Pipe through `jq` to get just the property we are looking for
+curl \
+  -X GET \
+"$GROUPAROO_HOST/api/v1/run/$RUN_ID?apiKey=$GROUPAROO_API_KEY" \
+| jq '.run.state'
+```
+
+Note that your API Key will need read access to the `run` permission as well.
+
+## Triggering Schedules via a Query
+
+> Coming soon.
+
+## Running Grouparoo directly via the CLI
+
+Grouparoo can always be run via the command line with `grouparoo run`. Some customers may want to run Grouparoo via the CLI directly from their orchestrator as the final part of their ETL/ELT pipeline, or simply every night via `cron`. For these use cases, we recommend configuring Grouparoo via code config and using the `grouparoo run` command. This will run all Schedules immediately when `grouparoo run` is started.
```

Comment:
```suggestion
Grouparoo can always be run via the command line with `grouparoo run`. Some customers may want to run Grouparoo via the CLI directly from their orchestrator as the final part of their ETL/ELT pipeline, or simply every night via `cron`. For these use cases, we recommend configuring Grouparoo via [code config](/docs/config/code-config) and using the `grouparoo run` command. This will run all Schedules immediately when `grouparoo run` is started.
```